.pre-commit-config.yaml

```diff
@@ -8,8 +8,8 @@ repos:
         files: ^infra/
         args:
           - --args=--exclude-downloaded-modules
-      - id: terraform_validate
-        args:
-          - --init-args=-lockfile=readonly
-          - --args=-json
-          - --args=-no-color
+      # - id: terraform_validate
```

Comment:
why this change?

Reply:
The newly introduced repository structure with modules and a subfolder <environment>/<region>/ is causing this pre-commit check to fail. 

This check appears to be flagging the different versions of the azurerm provider requested by each module, which imho contradicts the principle of reusable modules.

Together with @Krusty93, we are investigating the issue, but no solution has been found yet. The idea is to enable it back when a solution is found. What do you think?

Reply:
it's ok, but I'd add this information as a short comment inside.